forc-plugins/forc-client/src/op/call/missing_contracts.rs

```diff
@@ -45,7 +45,7 @@ pub async fn get_missing_contracts(
         {
             Ok(_) => return Ok(call.external_contracts),
             Err(fuels_core::types::errors::Error::Transaction(
-                fuels::types::errors::transaction::Reason::Reverted { receipts, .. },
+                fuels::types::errors::transaction::Reason::Failure { receipts, .. },
             )) => {
                 let contract_id = find_id_of_missing_contract(&receipts)?;
```

Comment:
nit: we may also be able to use the function from the rust sdk now: https://github.com/FuelLabs/fuels-rs/blob/dbc896f97765f1db5a30ac1d51b0ba69baa448aa/packages/fuels-programs/src/calls/utils.rs#L325
and with that, we could remove the implementation below.

Reply:
could be another PR though; since its unrelated to the bump

Reply:
I think it is better to keep this one focused only to bump. We can create an issue for it.